# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/shadow/pamd-default_config.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./etc/pam.d/login.orig	2004-12-23 19:20:50.124013792 +0100
+++ ./etc/pam.d/login	2004-12-23 19:21:14.542301648 +0100
@@ -1,6 +1,6 @@
 #%PAM-1.0
 # /etc/pam.d/login
-auth            required        pam_securetty.so
+#auth           required        pam_securetty.so
 auth            required        pam_unix.so
 auth            required        pam_tally.so file=/var/log/faillog onerr=succeed no_magic_root
 auth            required        pam_shells.so
@@ -12,8 +12,7 @@
 account         required        pam_unix.so
 password        required        pam_cracklib.so difok=2 minlen=8 dcredit=2 ocredit=2 retry=3
 password        required        pam_unix.so md5 shadow use_authtok
-password        required        pam_make.so /var/db
 session         required        pam_unix.so
 session         required        pam_env.so
 session         required        pam_limits.so change_uid
-session         optional        pam_console.so
+#session        optional        pam_console.so
--- ./etc/pam.d/chage.orig	2004-12-23 19:21:21.602228376 +0100
+++ ./etc/pam.d/chage	2004-12-23 19:21:27.754293120 +0100
@@ -2,4 +2,3 @@
 # /etc/pam.d/chage
 auth		sufficient	pam_rootok.so
 account		required	pam_permit.so
-password	required	pam_make.so /var/db
--- ./etc/pam.d/chpasswd.orig	2004-12-23 19:21:32.728536920 +0100
+++ ./etc/pam.d/chpasswd	2004-12-23 19:21:36.665938344 +0100
@@ -2,4 +2,3 @@
 # /etc/pam.d/chpasswd
 auth		sufficient	pam_rootok.so
 account		required	pam_permit.so
-password	required	pam_make.so /var/db
--- ./etc/pam.d/groupadd.orig	2004-12-23 19:21:41.124260576 +0100
+++ ./etc/pam.d/groupadd	2004-12-23 19:21:44.268782536 +0100
@@ -2,4 +2,3 @@
 # /etc/pam.d/groupadd
 auth		sufficient	pam_rootok.so
 account		required	pam_permit.so
-password	required	pam_make.so /var/db
--- ./etc/pam.d/groupdel.orig	2004-12-23 19:21:49.481990008 +0100
+++ ./etc/pam.d/groupdel	2004-12-23 19:21:52.520528080 +0100
@@ -2,4 +2,3 @@
 # /etc/pam.d/groupdel
 auth		sufficient	pam_rootok.so
 account		required	pam_permit.so
-password	required	pam_make.so /var/db
--- ./etc/pam.d/groupmod.orig	2004-12-23 19:22:00.196361176 +0100
+++ ./etc/pam.d/groupmod	2004-12-23 19:22:02.988936640 +0100
@@ -2,4 +2,3 @@
 # /etc/pam.d/groupmod
 auth		sufficient	pam_rootok.so
 account		required	pam_permit.so
-password	required	pam_make.so /var/db
--- ./etc/pam.d/newusers.orig	2004-12-23 19:22:11.764602536 +0100
+++ ./etc/pam.d/newusers	2004-12-23 19:22:15.149088016 +0100
@@ -2,4 +2,3 @@
 # /etc/pam.d/newusers
 auth		sufficient	pam_rootok.so
 account		required	pam_permit.so
-password	required	pam_make.so /var/db
--- ./etc/pam.d/usermod.orig	2004-12-23 19:22:20.528270256 +0100
+++ ./etc/pam.d/usermod	2004-12-23 19:22:23.536812888 +0100
@@ -2,4 +2,3 @@
 # /etc/pam.d/usermod
 auth		sufficient	pam_rootok.so
 account		required	pam_permit.so
-password	required	pam_make.so /var/db
--- ./etc/pam.d/passwd.orig	2005-12-27 21:06:57.000000000 +0100
+++ ./etc/pam.d/passwd	2005-12-27 21:07:04.000000000 +0100
@@ -1,3 +1,3 @@
 #%PAM-1.0
 # /etc/pam.d/passwd - sample PAM config file for the `passwd' service
-password	required	pam_unix_passwd.so
+password	required	pam_unix.so
